Update docs to include Enterprise Plan

bturcotte520 opened the pull request:
Updating docs to include Enterprise Plan

## Context

Kilo for Enterprise was missing from Docs, adding several sections and added to sidebar menu

## Implementation

Adapted Kilo for Teams documentation and updated it with new features

bturcotte520 committed:
Create getting-started guide for Kilo for Enterprise

Added a comprehensive guide for getting started with Kilo for Enterprise, detailing features, setup steps, and support options.

bturcotte520 committed:
Update roles and permissions documentation for enterprise

bturcotte520 committed:
Add dashboard overview documentation for Kilo Enterprise

bturcotte520 committed:
Add team management documentation

This document provides guidelines for managing teams in Kilo for Enterprise, including adding members, managing roles, setting usage limits, and troubleshooting common issues.

bturcotte520 committed:
Add billing and credits documentation

This document outlines the billing and credits system for Kilo for Enterprise, detailing subscription costs, AI credits, management processes, and troubleshooting steps.

bturcotte520 committed:
Add usage analytics documentation for Kilo for Enterprise

Added detailed usage analytics documentation for Kilo for Enterprise, covering dashboard overview, summary metrics, time period filters, usage view options, data breakdown views, and cost tracking.

bturcotte520 committed:
Adding images to getting started

bturcotte520 committed:
Adding images

bturcotte520 committed:
Adding images

bturcotte520 committed:
Add Kilo for Enterprise category to sidebars

alexkgold commented on apps/kilocode-docs/docs/enterprise/roles-permissions.md:
```diff
@@ -0,0 +1,183 @@
+- Following team policies and guidelines
+- Productive AI-assisted development
+
+## Detailed Permissions Matrix
```
Audit logs?

alexkgold reviewed (changes requested):
Hi @bturcotte520 -- this is a fine start, but I don't yet see the parts we need the most -- documentation of the features that aren't yet documented (all Enterprise-only features + organization modes).

So we really need pages on:
* Audit Logs
* Model access
* Custom Modes
* SSO

Those will require a little experimentation and digging, because they aren't documented at all right now. I can help you understand the features, or you can talk to Brian C or John F who have worked on those features.

bturcotte520 committed:
Merge branch 'main' into bturcotte520-patch-1

bturcotte520 committed:
Add SSO setup documentation for Kilo for Enterprise

Document the process of enabling Single Sign-On (SSO) for Kilo for Enterprise, including prerequisites and setup steps.

bturcotte520 committed:
Add documentation for Model Access feature

Document the Model Access feature for organization admins, detailing management, filtering options, use cases, and important notes.

bturcotte520 committed:
Add documentation for Audit Logs feature

This document outlines the purpose and usage of Audit Logs, including how to view and filter logs, as well as details about log entries.

bturcotte520 committed:
Move SSO.md to the enterprise seats directory

bturcotte520 committed:
Move SSO.md to the enterprise directory

bturcotte520 committed:
Add Custom Modes documentation

Added documentation for creating and managing Custom Modes in Kilo.

bturcotte520 committed:
Move SSO.md to the enterprise seats directory

bturcotte520 committed:
Move analytics.md to seats directory

bturcotte520 committed:
Move audit-logs to seats/enterprise

bturcotte520 committed:
Move billing.md to /seats

bturcotte520 committed:
Move to enterprise

bturcotte520 committed:
Move getting-started.md to seats/enterprise directory

bturcotte520 committed:
Rename apps/kilocode-docs/docs/enterprise/model-access.md to apps/kilocode-docs/docs/seats/enterprise/model-access.md

bturcotte520 committed:
Move roles-permissions.md to seats directory

bturcotte520 committed:
Move team management documentation to seats directory

bturcotte520 committed:
Move getting-started.md to seats/teams directory

bturcotte520 committed:
Rename dashboard.md to seats/teams/dashboard.md

bturcotte520 committed:
Delete apps/kilocode-docs/docs/teams directory

bturcotte520 committed:
Update documentation for Kilo for Enterprise & Teams

bturcotte520 committed:
Revise billing details for Kilo Seats

Updated billing information to reflect 'Kilo Seats' instead of 'Kilo for Enterprise' and clarified subscription details.

bturcotte520 committed:
Update custom-modes.md

bturcotte520 committed:
Update role system description for Kilo Seats

bturcotte520 committed:
Update sidebar with Kilo Seats items

bturcotte520 committed:
update migration guide for Kilo AI coding tools

Add comprehensive migration guide for switching to Kilo from other AI coding tools, including benefits, processes, and training.

bturcotte520 committed:
Merge branch 'main' into bturcotte520-patch-1

bturcotte520 committed:
Update getting-started.md with sign-up details

Clarified sign-up options and added a note about using a company email for enterprise accounts.

bturcotte520 committed:
Add 'View audit logs' permission to roles table

alexkgold committed:
Add Alex Edits

alexkgold committed:
Merge branch 'main' into bturcotte520-patch-1

alexkgold committed:
Update email documentation

alexkgold committed:
Merge branch 'bturcotte520-patch-1' of github.com:Kilo-Org/kilocode into bturcotte520-patch-1

alexkgold committed:
Fix links
